docs/website/data/integrations.yaml

```diff
@@ -167,6 +167,8 @@ integrations:
       layer: network
     code:
       - https://github.com/build-security/opa-express-middleware
+    inventors:
+      - build.security      
```

Comment:
Change `build.security` to `build-security` in the `inventors` section.

Reply:
I did the inverse (changed the organization to be build.security)